lametro/events.py

```diff
@@ -87,6 +90,9 @@ def api_events(self, *args, **kwargs):
                 partner_event = self._find_partner(unpaired_event)
                 if partner_event is not None:
                     yield partner_event
+                else:
+                    logging.warning('No Spanish/English match found for {0} {1} {2}'.format(
```

Comment:
`_find_partner` should break if it cannot pair a spanish event. so, we should only reach this block if we have an unpaired english event. how about updating the logging message to `Cannot find Spanish companion for {EventBodyName} at {EventTime} on {EventDate}. {api_url}`? that way we have a record of the important metadata at the time of the scrape, and a link for further investigation.